lms/resources/_js_config.py

```diff
@@ -167,7 +167,10 @@ def google_picker_origin():
             # When we're being launched in an iframe within the LMS our JavaScript
             # needs to pass this URL (which is the URL of the top-most page) to Google
             # Picker, otherwise Picker refuses to launch inside an iframe.
-            return self._context.custom_canvas_api_domain or self._ai_getter.lms_url()
+            return (
+                self._context.custom_canvas_api_domain
+                or self._application_instance_service.get().lms_url
```

Comment:
So this would become:

```suggestion
                or self._application_instance().lms_url
```

etc.

Reply:
:+1: